dosagelib/plugins/s.py

```diff
@@ -267,12 +267,12 @@ class ShotgunShuffle(_BasicScraper):
 
 class SinFest(_BasicScraper):
     description = u'Strip dealing with contemporary issues and religion. Created by Tatsuya Ishida.'
-    name = 'KeenSpot/SinFest'
+    name = 'SinFest'
```

Comment:
Please don't change the name. Otherwise users who might have started with "KeenSpot/SinFest" can't update anymore...